submissions/benjaminKamena/html/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Contact page</title>
+</head>
+<body>
+    <div>
+        <h2>Contact Us</h2>
+        <p>Swing by for more products, or leave us a message:</p>
+    </div>
+    <div>
+        <form>
+            <label for="fullName">Full Name</label><br>
+            <input type="text" id="fullName" name="fullName" placeholder="Your name.."><br><br>
+            <label for="email">Enter Email Address</label><br>
+            <input type="text" id="email" placeholder="Email"><br><br>
+            <label for="subject">Subject</label><br>
```

Comment:
@kamena1994 Well done! I think you missed a step here. There is input field for subject and the message as well. You need to seperate the two entities.